terraform/environments/analytical-platform-ingestion/dms/dms.tf

```diff
@@ -0,0 +1,57 @@
+locals {
+    short_resource_name = "tariff-${local.environment}"
+    resource_name = "cica-ap-${local.short_resource_name}"
+}
+
+module "cica_dms_tariff_dms_implementation" {
+
+    source      = "../modules/dms"
+    vpc_id      = data.aws_vpc.connected_vpc.id
+    environment = local.environment
+
+    db          = local.short_resource_name
+
+    dms_replication_instance = {
+        replication_instance_id    = local.resource_name
+        subnet_cidrs               = local.environment_configuration.connected_vpc_private_subnets
```

Comment:
This should use a data call to get the subnet CIDRs, not hardcoded environment configurations

Reply:
Yeah I can make that change if you want. I guess the precedent of `analytical-platform-ingestion/environment-configuration.tf` doesn't really apply here as this isn't being used to define a resource